openthread_border_router/config.yaml

```diff
@@ -37,9 +37,10 @@ ports_description:
   8080/tcp: OpenThread Web port
   8081/tcp: OpenThread REST API port
 schema:
-  device: device(subsystem=tty)
+  device: device(subsystem=tty)?
```

Comment:
So far device was mandatory, making it optional will allow users to not set it. This means after installation, if people just press start, they'll end up in an error (as no network device and no tty is specified). I guess it will be `otbr-agent` running into an error because `device` is empty. Can you add a `bashio::config.require` for the device config (if network is not set) so the user gets a clear error what needs to be provided?

Reply:
This didn't actually work as expected, device is always mandatory, I suppose its somehow related to the `device=null` default.

I will just remove `?` for now.